mkdocs.yml

```diff
@@ -175,6 +178,7 @@ markdown_extensions:
   - attr_list
   - md_in_html
   - pymdownx.caret
+  - sane_lists
```

Comment:
I couldn't get mkdocs to render an ordered list which one of them having unordered items correctly without this.